framework/util/image_writer.cpp

```diff
@@ -320,6 +320,31 @@ static const uint8_t* ConvertIntoTemporaryBuffer(uint32_t    width,
         }
         break;
 
+        case kFormat_S8_UINT:
```

Comment:
Is this required for dumping specific subresources or is this a fix for a separate problem?

Reply:
I enable dumping of stencil aspect. This is in a separate commit in this PR